apps/web/src/app/events/[slug]/page.tsx

```diff
@@ -22,18 +22,43 @@ export async function generateMetadata({
   params: { slug: string };
 }): Promise<Metadata> {
   const event = await getEvent(params.slug);
+  const lectures = await getEventLectures(params.slug);
+
   if (!event) {
     notFound();
   }
+  lectures;
   let title = event.title;
   if (event.subtitle) {
     title += " " + event.subtitle;
   }
+
+  if (title.length > 55) {
+    title = title.substring(0, 55) + "...";
+  }
+
+  if (event.description.length > 100) {
+    event.description = event.description.substring(0, 100) + "...";
+  }
+
+  function getSpeakersNames() {
+    const speakers = [];
+    lectures.map((lecture) => {
+      lecture.speakers.map((speaker) => {
+        speakers.push(speaker.name);
+      });
+    });
+    return speakers;
+  }
+
   return {
+    metadataBase: new URL(`https://jeteo.newbies.pl/events/${event.slug}`),
```

Comment:
tu chyba pole jest zbędne, starczy to metadataBase uzupełnić na głównym layoucie, kojarzę że ustawiało się to tylko jakoś żeby dziedziczyły te metadane po sobie